test_scripts/Defects/6_1/885_For_some_RPCs_Core_does_not_unsubscribe_from_softbuttons_after_receiving_a_response/Alert.lua

```diff
@@ -0,0 +1,109 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/885
+-- Description:
+-- For some RPCs, Core does not unsubscribe from softbuttons after receiving a response
+-- Precondition:
+-- SDL and HMI are started
+-- In case:
+-- 1) App is registered and activated
+-- 2) App sends "Alert" request
+-- Expected result:
+-- 1) For all of the popup-based RPCs which contain softButtons, Core is supposed to stop processing softButton events once a response is received for the RPC.
+-- Currently the only one that does this is ScrollableMessage, for all of the others Core stays subscribed to their softButtons after receiving a response.
+-- Actual result:
+-- 1)
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function alert()
+    local params = {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
+        },
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        },
+        alertText1 = "alertText1",
+        progressIndicator = true
+    }
+    local responseDelay = 3000
+    local cid = common.getMobileSession():SendRPC("Alert", params )
+    EXPECT_HMICALL("UI.Alert",{
+        alertStrings = {
+            { fieldName = "alertText1", fieldText = "alertText1" }
+        },
+        duration = 0,
+        softButtons = {
+            { type = "TEXT",
+            softButtonID = 4 }
+        },
+        alertType = "BOTH",
+        appID = common.getHMIAppId(),
+        progressIndicator = true
+    })
+    :Do(function(_, data)
+        local function alertResponse()
+            common.getHMIConnection():SendResponse(data.id, "UI.Alert", "SUCCESS", { })
+        end
+        RUN_AFTER(alertResponse, responseDelay)
+    end)
+    params.appID = common.getHMIAppId()
+    common.getHMIConnection():ExpectRequest("TTS.Speak", {
+        ttsChunks = {
+          { type = "TEXT", text = "pathToFile" }
```

Comment:
Should tie this directly to `params.ttsChunks` rather than duplicating the value